app.py

```diff
@@ -95,6 +95,8 @@ def session_stats_create(session_key):
     session = Session.query.get(session_key)
     if session is None:
         session = Session(session_key)
+        session.client_updated_at = datetime.utcnow()
```

Comment:
Before:
- session was created during node discovery
Now:
- should we create session sooner than after minute?